config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,23 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func isMountpointForbidden(path string) bool {
+	// Check if the path is a restricted directory
+	restrictedDirs := []string{"/boot", "/sys", "/usr", "/var", "/", "/sysroot/mnt"}
```

Comment:
Hmm, it might be less of a footprint to be less explicit here. What if we flip the logic on its head, and only allow the allowed paths and their subpaths?

Looking at the [docs](https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_configuration_in_etc_and_state_in_var) we only allow `/etc` and `/var`

Reply:
Makes sense to do it that way, thanks!